### PR TITLE
Basic UI

### PR DESCRIPTION
**Pull request by prius2055**
### Basic UI

_**In this repository, I have implemented the following;**_

- Created IU in the command line window
- Created logic to allow a user to add book 
- Created logic to allow a user to add both students and teachers to the library directory
- Created logic to allow any person rent a book
- Created logic to track all rented books.
- Linters configuration

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  def list_people
+    people = library.list_people
+    puts 'List of people:'
+    if people.empty?
+      puts 'No people found.'
+    else
+      people.each { |person| puts "Name: #{person.name}, Age: #{person.age}, Id: #{person.id} " }
+    end
+  end
+
```
Comment (BenMukebo) [inline app.rb]: - [ ] Great job so far 👏🏻 But, your school library console app must behaves exactly  in the same was as the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) provided for you.
- Check the screen short bellow 👇🏻 


image

- 1 Make sure to dispaly the  list_books option (1)  as shown in the video. You need to specify the Title of the book and the Author of the book. Eg
```ruby
Tile: "...",  Author: "...."
```
- 2 Make sure you display the list_people option (1)  as shown in the video. You need to specify the  if is the Student or Teacher.
```ruby
[Student] Name: Rose, ...
[Teacher] Name: ....

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @prius2055,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- Professional README ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (prius2055) [reply app.rb]: This was an oversight. It has been addressed.Thank you

**Review by laiifuu — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: 🎉

## Hi @prius2055 :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

friends-joey


## Highlights
- [x] Implemented the required changes :heavy_check_mark: 
- [x] Organized code :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**